lib/features/caching/caching_manager.dart

```diff
@@ -64,6 +70,8 @@ class CachingManager {
       _fcmCacheClient.clearAllData(),
       _firebaseRegistrationCacheClient.clearAllData(),
       _recentSearchCacheClient.clearAllData(),
+      _recentLoginUsernameCacheClient.clearAllData(),
+      _recentLoginUrlCacheClient.clearAllData(),
```

Comment:
Should not clear these `table`, we need it for suggesting people recently account

Reply:
- We will use SharePreferences to store recent object

Reply:
I used another solution for this problem. Please review it. Thanks